PACKAGING.md

```diff
@@ -25,8 +25,10 @@ A release is a [git tag](https://git-scm.com/docs/git-tag) in the `main` branch
 
 ### AUR
 
-[ianny-git](https://aur.archlinux.org/packages/ianny-git)
+- [ianny](https://aur.archlinux.org/packages/ianny)
+- [ianny-git](https://aur.archlinux.org/packages/ianny-git)
+- [ianny-bin](https://aur.archlinux.org/packages/ianny-bin)
```

Comment:
```suggestion
- [ianny-git](https://aur.archlinux.org/packages/ianny-git)
```

This list should be just for official packages to ack like an official reference for anyone who wan to create a new package.